### DISCUSSION TITLE
Data e Ora

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1387] [HUB: Home Assistant]
Ciao a tutti, sto cercando di definire la mia Plancia e sto visionando diverse immagini per capire le varie potenzialità.
Molte plance riportano data e ora, non sono riuscito a trovare nulla che mi riportasse questi dati in plancia, come posso fare? c'è qualche repository che si può utilizzare? grazie mille

**Comment by dante70** [comment]
Ciao,io ho usato una button_card che ho personalizzato con colori e bordo a mio piacimento. 
Trovi il componente su HACS

**Reply by mriitan** [reply]
mi inseriresti la tua card così da personalizzarla? grazie

**Comment by dante70** [comment]
```
  - type: custom:button-card
   
    entity: sensor.time
    layout: vertical
    label: '[[[ return states[''sensor.temperatura_esterna''].state; ]]]'
    name: '[[[ return states[''sensor.pretty_date''].state; ]]]'
    show_icon: false
    show_label: FALSE
    show_name: true
    show_state: true
    styles:
          card:
            - width: 180px
            - height: 75px
            - margin-left: 5px
          #  - margin-top: 10px
            - border-radius: 5px
            - border-width: 1px
            - border-style: solid
            - border-color:  grey
            
            - background-color: rgba(230,230,230)  # (--primary-background-color)
          icon:
            - color: black       
       
          state:
            - font-size: 200%
            - font-weight: bold
            - justify-self: center
          label:
            - justify-self: center
            - font-size: 90%
        
          name:
             - justify-self: center
             - font-size: 80%
             - position: top
    tap_action:
      action: none

**Reply by mriitan** [reply]
perdonami ma disponi di un sensor.time, io non ho quella entità

**Reply by dante70** [reply]
https://www.home-assistant.io/integrations/time_date/

**Reply by mriitan** [reply]
esatto, grazie quello che mi serviva, m dove c'è un elenco di tutte le funzioni applicabili?
grazie

**Reply by dante70** [reply]
le funzioni applicabili? intendi come personalizzare la card?

**Reply by mriitan** [reply]
intendo tutte le funzioni per personalizzare le card, ho cercato sulla documentazione e parlava di calendar, ma queste funzioni non venivano riportate.

**Reply by dante70** [reply]
per personalizzare la button card devi vedere https://github.com/custom-cards/button-card 
il mio consiglio e' quello di leggere ,leggere e provare .Google ti aiuta e poi sei tu che devi buttarti e provare.
